src/main/scala/org/renci/chemotext/Main.scala

```diff
@@ -28,6 +28,34 @@ import akka.stream._
 import akka.stream.scaladsl._
 import io.scigraph.annotation.EntityAnnotation
 import io.scigraph.annotation.EntityFormatConfiguration
+import org.apache.jena.datatypes.xsd.XSDDatatype
+
+import scala.collection.{SortedSet, immutable}
+
+case class ArticleInfo(pmid:String,info:String,meshTermIDs:Set[String],years:Set[String])
```

Comment:
I like to have spaces after commas. IntelliJ autoformat should do this automatically.

Reply:
Are you asserting more than one year using the same predicate? Will that be weird?